frontend/pages/index.vue

```diff
@@ -94,10 +117,10 @@ export default {
 .intro-section {
   background-color: #185262;
   background-image: url(../images\trisight-background-home-desktop.png);
```

Comment:
Did you remove the background image? I don't see it anymore: 
![image](https://github.com/user-attachments/assets/67a92860-d100-4d38-ad00-d98d62cfc399)